join.c

```diff
@@ -117,20 +118,27 @@ void joinFreeCallback(void *privdata)
  */
 void joinIdleCallback(Connection *conn)
 {
+    RedisRaftCtx *rr = ConnGetRedisRaftCtx(conn);
     JoinState *state = ConnGetPrivateData(conn);
 
+    time_t now;
+    time(&now);
+
+    // This value should probably be dynamic and be considered a multiple of election time period? 2x?
```

Comment:
@sjpotter Current convention is c-style `/**/` comments, let's stick with it.
I think x2 or x3 the election period seems right - and we should keep this constant defined in `redisraft.h` and not hidden here.